resources/public/api/conf/1.0/lossClaims_amend.raml

```diff
@@ -0,0 +1,25 @@
+is:
+  - acceptJson
+  - contentTypeJson
+  - testScenarioHeader
+  - formatNino
+  - formatClaimId
```

Comment:
"... loss claim ID..." vs "... loss claim id...". We use loss id in the BF endpoints it seems.

Reply:
We've also used ID though ("self employment ID"). I'm not sure which is correct but I can make it consistent I guess. Should ID be capitalised or not?